### DISCUSSION TITLE
"Follow" links in the (left) sidebar does not show on iPad  with Safari.

### DISCUSSION THREAD

**Discussion by makotonakajima** [mmistakes/minimal-mistakes#2684] [Q&A] [answered]
Hi,

Thank you very much for creating this awesome theme. Since I don't have any experience with Github, and know only a little bit of html/css, my question might be trivial. But here we go.

I just forked mmistakes/minimal-mistakes and started re-configuring (for an academic homepage) a week ago and noticed that the "Follow" links of the author info on the (left) side-bar does now show up when viewing the page with my iPad pro (with Safari). The screen size is 2224-by-1668. When I use the screen horizontally, the photo and the description below the photo shows up on the left (like in any large screen), but the list of links below them do not show up. If I use the screen in the portrait direction, the "Follow" button shows up, but nothing happens if I push it. The page that I am working on is here:

https://github.com/makotonakajima/makotonakajima.github.io

I thought I screwed up something when I was tweaking parameters, but the same thing happens with, for example, here:

https://mmistakes.github.io/minimal-mistakes/year-archive/

So this is not a problem of the tweaked version of mine. I tried my old iPad with an old (outdated) Safari and screen size of 2048 x 1536, and there is no problem. Since both have a larger screen than the max size of the theme, I suspect it is related to the (most) recent version of Safari. I suspect this is related to how Breakpoints work, together with the size of the screen obtained from the browser, but I don't have any deep knowledge in terms of how this work. Any help would be appreciated.

Thank you again for this great theme.

Makoto

**Comment by mmistakes** [comment, marked as answer]
Do you have any content blockers enabled in Safari? I know in Chrome when using privacy/adblocker plugins they are a bit over zealous and remove the social links.

**Comment by makotonakajima** [comment]
Bingo! I am so ashamed that I didn't think about it. Yes, I had AdGuard with various custom filters, and the problem went away when I turned them off. I didn't think about it because various content blockers of Chrome on my PC did not cause a problem. Thank you so much for a quick help!

**Comment by mmistakes** [comment]
No problem. No idea why content blockers go after Font Awesome icons, but for whatever stupid reason they kill the icons and the text/links near them.
